templates/case-studies/templates/_macros-hero.jinja

```diff
@@ -0,0 +1,31 @@
+# Params
+# title: Hero title text (required)
+# subtitle: Hero subtitle text
+# logo: Path to logo image
+# logo_alt: Alt text for logo image
+# logo_width: Width of logo image (128 by default)
+# logo_height: Width of logo image (128 by default)
+
+{%- macro hero(
+  title,
+  subtitle,
+  logo,
+  logo_alt="",
+  logo_width="128",
+  logo_height="128"
+  ) -%}
+  <section class="p-section--hero">
+    <div class="row--25-75">
+      <div class="col">
+        <div class="p-image-wrapper">
+          <img src={{ logo }} alt="{{ logo_alt }}" width="{{ logo_width }}" height="{{ logo_height }}" />
+        </div>
+      </div>
+      <div class="col">
+        <h1>{{ title }}</h1>
```

Comment:
Looks like this heading should be wrapped in a `p-section--shallow` div as well 
![image](https://github.com/user-attachments/assets/666d9429-1f89-4f1b-94a6-cc4488387507)